libs/portals/my-pages/occupational-licenses/src/screens/OccupationalLicensesOverview/OccupationalLicensesOverview.tsx

```
@@ -1,42 +1,144 @@
+                    label: formatMessage(m.view),
+                    variant: 'text',
+                    url: OccupationalLicensesPaths.OccupationalLicensesDetail.replace(
+                      ':id',
+                      license.licenseId,
+                    ),
+                  }}
+                  image={{
+                    type: 'image',
+                    url: image,
+                  }}
+                />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)